tests/test_data.yml

```diff
@@ -1,5 +1,8 @@
 ---
 facebook:
+  avatar_usernames:
```

Comment:
why do we need different usernames for facebook?

Reply:
As noted in the PR description, profiles on Facebook can be “hidden”, therefore having no avatar to display. These are public profile links.

Reply:
ok